docs/deprecated.md

```diff
@@ -20,6 +20,17 @@ The following list of features are deprecated in Engine.
 To learn more about Docker Engine's deprecation policy,
 see [Feature Deprecation Policy](https://docs.docker.com/engine/#feature-deprecation-policy).
 
+### Unencrypted password storage via `docker login`
+
+**Deprecated In Release: v17.10.0**
+
+**Target For Removal In Release: v18.10**
+
+Docker before v17.10 stored user credentials in `~/.docker/config.json`
+unencrypted by default. As of v17.10 it prints a warning with a link to the
+credentials helper page and asks users if they want to continue. Eventually, we
```

Comment:
I'd skip the niceties and just, e.g.: "Support for unencrypted password storage will be removed in Docker 18.10."